recipes/civictheme_site_install/recipe.yml

```diff
@@ -30,6 +30,8 @@ config:
     media_library:
       - core.entity_form_mode.media.media_library
       - core.entity_view_mode.media.media_library
+      - image.style.media_library
+      - views.view.media_library
```

Comment:
This fixes a fatal error when you click on the "Add Media" button on media fields.

The media library view wasn't being imported.